### PR TITLE
feat: project-form-modal

### PR DESCRIPTION
**Pull request by rupertbright-hub**
Modal added for new project form (no close feature)

### CODE REVIEW

File: client/src/Context.tsx [hunk 1]
```diff
@@ -3,14 +3,16 @@ import { createContext } from 'react';
 type State = {
   currentProjectId: number;
   userId: string;
-  isModalOpen: boolean;
+  isBugOpen: boolean;
```
Comment (ilyadusoleil) [inline client/src/Context.tsx:6]: personally I'd leave this as isBugModalOpen, as isBugOpen could have a wider range of meanings

File: client/src/components/ProjectHeader.tsx [hunk 2]
```diff
@@ -70,7 +70,7 @@ const ProjectHeader = () => {
         <div
           className="h-10 flex items-center"
           onClick={() => {
-            navigate('/newProject');
+            ctx.dispatch({ type: 'openProjectModal'});
```
Comment (ilyadusoleil) [inline client/src/components/ProjectHeader.tsx:73]: And would probably make sense to collapse the header too as well

**Review by ilyadusoleil — CHANGES_REQUESTED** [review]
Looks good! Update the variable names and then complete the pull request 👍

**Review by rupertbright-hub — COMMENTED** [review]
variable names changed. Collapsing of header on click of new project to be completed.

**Pull request merged**